src/data/schema/user.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE users
+(
+	user_id SERIAL NOT NULL
+		CONSTRAINT users_pk
+			PRIMARY KEY,
+	name TEXT,
+	email TEXT NOT NULL,
+	phone TEXT,
+	password TEXT,
+	major TEXT,
+	degree TEXT,
+	enable BOOLEAN DEFAULT TRUE
+);
+
+ALTER TABLE users owner TO postgres;

```

Comment:
Do you mean the 'enable' attribute? This one marks the user is in use, and it would be set false after deleting the user. In this case, we are able to collect user information cause we don't delete it.